source/_components/vera.markdown

```diff
@@ -46,3 +48,6 @@ vera:
   lights: [15, 17, 19, 21, 22, 24, 26, 43, 64, 70, 87]
 ```
 
+### {% linkable_title Using z-wave devices in automation %} 
+
+If you want to use a z-wave device from the Vera controller in Home Assistant automation, you'll need the entity id. In the Home Assistant UI you'll find all entities listed under the <img src='/images/screenshots/developer-tool-states-icon.png' alt='service developer tool icon' class="no-shadow" height="38" /> icon of the Developer Tools section. Look for entities that contain 'Vera Device Id' in their attributes, you'll find the entity id on the left.
```

Comment:
You are correct. I had fixed the spelling in an older piece of text but failed to write it correctly in my own :-|